src/amo/components/InstallButtonWrapper/index.js

```diff
@@ -35,7 +29,6 @@ export type Props = {|
   addon: AddonType,
   className?: string,
   defaultButtonText?: string,
-  getFirefoxButtonType: GetFirefoxButtonTypeType,
```

Comment:
`WithInstallHelpersInjectedProps` should be removed from `InternalProps`.

Reply:
As described above, I have removed this.